.github/workflows/bumpup.yml

```diff
@@ -0,0 +1,51 @@
+name: Bump up gem version
+
+on:
+  workflow_dispatch:
+    inputs:
+      gem-version:
+        description: Gem version
+        required: true
+        type: string
+      email:
+        description: Your email
+        required: true
+        type: string
+
+jobs:
+  bump_up_gem_version:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+      with:
+          fetch-depth: 0
+
+    - uses: ruby/setup-ruby@v1
+      with:
+        ruby-version: '3.1'
+        bundler-cache: true
+
+    - name: config
+      run: |
+        git config --global user.email ${{ github.event.inputs.email }}
+        git config --global user.name ${{ github.actor }}
```

Comment:
`github-actions[bot]` is available.
ref. https://github.com/actions/checkout?tab=readme-ov-file#push-a-commit-using-the-built-in-token

If you use `github-actions[bot]`, you will not need to enter email address.

Reply:
Fix: [d25ea67](https://github.com/kufu/jpostcode-rb/pull/567/commits/d25ea674dc5496818391ef954155147dce0d481c)